docs/python-grader/index.md

```diff
@@ -230,6 +230,34 @@ Some courses may use libraries that are common across multiple questions. For su
 }
 ```
 
+
+
+### Example Usage of `serverFilesCourse` for static data
+
+The Python autograder is able to retrieve information from `serverFilesCourse`. This can be used when there are files and other static data that can be shared across multiple questions. For example, assume there is a data file called `chem.json` used in multiple questions. The file can be saved in the `serverFilesCourse` directory within the course root directory, for example at `serverFilesCourse/compounds/chem.json`.
+
+To access `serverFilesCourse` from the autograder, specify the file or its containing directory in the question `info.json`. For example, to copy the `compounds` directory to the autograder, use:
+
+```diff
+...
+    "externalGradingOptions": {
+        "enabled": true,
+        "image": "prairielearn/grader-python",
+        "entrypoint": "/python_autograder/run.sh",
++       "serverFilesCourse": ["compounds"]
+    }
+...
+```
```

Comment:
I'm not that convinced that a diff-style block would be a good idea here. While highlighting the proper line looks interesting, many users copy-paste these codes, so having the + there may cause issues. @mwest1066 thoughts? This is what it looks like in readthedocs:

![image](https://github.com/user-attachments/assets/626019cb-674b-4105-bbb8-aedadb33cbec)

Reply:
Can we highlight a line within a code block? It would be better. I thought about it but I did know we have that option.

Reply:
I'm not convinced it's that necessary, note that other code segments in the docs don't use it and the message still comes across.